lms/djangoapps/courseware/views/index.py

```diff
@@ -436,6 +442,10 @@ def _create_courseware_context(self):
                 table_of_contents['previous_of_active_section'],
                 table_of_contents['next_of_active_section'],
             )
+            # sections can hide data that masquerading staff should see when debugging issues with specific students
+            section_context['specific_masquerade'] = (
+                self.is_staff and self._is_masquerading_as_student(require_specific=True)
```

Comment:
Remove check for `is_staff` here since it's redundant.  `_is_masquerading_as_student` will never return `True` if `is_staff` is `False`.